pkgs/by-name/ed/edmarketconnector/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3,
+  makeWrapper,
+}:
+let
+  pythonEnv = python3.buildEnv.override {
+    extraLibs = with python3.pkgs; [
+      tkinter
+      requests
+      pillow
+      (watchdog.overrideAttrs {
+        disabledTests = [
+          "test_select_fd"
+        ];
```

Comment:
Any reason why you needed to disable the test? A comment for future readers would be useful I think.